conf/application-json-logger.xml

```diff
@@ -8,6 +8,7 @@
     <logger name="uk.gov" level="${logger.uk.gov:-INFO}"/>
 
     <logger name="reactivemongo.core.protocol.MongoHandler" level="${logger.reactivemongo:-WARN}"/>
+    <logger name="reactivemongo.core.actors.MongoDBSystem" level="${logger.reactivemongo:-WARN}"/>
```

Comment:
This one shows IsMaster responses, which we currently don't see.
Adding just reactivemongo.core, would also show lots of netty shaded logs... which aren't useful at present.